packages/toggleBox/style.ts

```diff
@@ -0,0 +1,6 @@
+import { css } from "emotion";
+
+export const style = css`
+  color: #000;
+  background-color: red;
```

Comment:
What is this?

Reply:
Something that gets automatically generated by `npm run create:component`. I will remove it